roles/telemetry_autoscaling/tasks/creating_stack.yml

```diff
@@ -82,7 +82,7 @@
     # source ~/overcloudrc;
     {{ openstack_cmd }} alarm list
   register: result
-  failed_when: result.rc >= 1
+  failed_when: '"cpu_alarm_high" not in result.stdout or "cpu_alarm_low" not in result.stdout'
```

Comment:
The original didn't do much checking. I think this condition is better as it does what's written in the task name.